### PR TITLE
More Productive Gestures

### PR DESCRIPTION
**Pull request by jazzhpatel**
Hey,

So i made some commits to the gestures implementation and added a single finger gesture which allows you to move the caret and the two finger gesture moves the caret a little slower ... i think this would completely change how we use the touch bar ... cuz now you wouldn't have to sacrifice pinky finger health to press arrow keys :)

Please try it and lemme know if this feels better

### CODE REVIEW

File: appcast.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" standalone="yes"?>
```
Comment (Toxblh) [inline appcast.xml:1]: and to del that file

File: README.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+
+# Better Gestures Edit!
+
+the commits to the code to get that make the cursor move by sliding on the touchbar ... you know how you could slide on the space bar on your phone ... it’s the same thing but on the touchbar and if you press ALT, it’ll scroll through words and if you press SHIFT, it’ll select the text, like you could move through tabs, just by holding CMD+SHIFT .... and two fingers makes it go slower for precision control ... its just single finger and double finger gestures sending left and right key presses.
+brooooooooooooooo
```
[the remark this answers is not in the document]
Comment (jazzhpatel) [reply README.md]: Gotcha, I made the readme a little more formal ... also, I changed menu item that said 'volume/brightness gestures' to 'default swipe gestures' so if the user wants to implement custom gestures, they could toggle that off ...

**Comment by jazzhpatel** [comment]
> It will be so good if don't need to scroll "central" part, which use "default" 1-finger scroll. But better don't reimplement legacy, and use it already like your configuration. And just stay to be able to add some custom behaviour for single finger scroll

I beg your pardon, I didn't entirely get that ... from what I understand the user could toggle the 'legacy' gestures from the menu if they wish to add their own functionality ...

**Comment by Toxblh** [comment]
I'll a bit facelift your code after merge.

**Pull request merged**